.github/workflows/test.yml

```
@@ -735,3 +735,13 @@ jobs:
           files: ./packages/generator-helper/src/__tests__/coverage/clover.xml
           flags: generator-helper,${{ matrix.os }},library,binary
           name: generator-helper-${{ matrix.os }}
+
+      - name: Upload test results to BuildPulse for flaky test detection
+        if: '!cancelled()' # Run this step even when the tests fail. Skip if the workflow is cancelled.
```

:wave: @janpio: If I'm reading the workflow correctly, this job runs on macOS and Windows:

https://github.com/prisma/prisma/blob/89cf96728993b1d06102317170956cf9345f2408/.github/workflows/test.yml#L587-L590

Given that there's [not yet a Windows binary for the BuildPulse test-reporter](https://github.com/Workshop64/buildpulse-action/issues/2#issuecomment-1047229576), how would you feel about adjusting this to be macOS-only for now?

I think the syntax will be something like this:

```suggestion
        if: ${{ runner.os == 'macos-11' && !cancelled() }} # Run this step even when the tests fail. Skip if the workflow is cancelled.
```

---

Reasonable suggestion to get it more working for now 👍